opentasks/src/main/java/org/dmfs/tasks/model/ContentSet.java

```diff
@@ -336,13 +340,13 @@ private ContentValues ensureAfter()
             values = new ContentValues();
             mAfterContentValues = values;
             // also create mAfterKeys
-            mAfterKeys = new HashSet<String>();
+            mAfterKeys = new HashSet<>();
```

Comment:
Please remove `mAfterKeys` while you're at it. It was only a workaround for the absence of `ContentValues.keySet()` on Android 2.x. Since we don't support Android 2 any more you can remove it in most places and replace by `mAfterContentValues.keySet()` where it's being read.

Reply:
Ok, I've removed `mAfterKeys`.